src/codeflare_sdk/ray/cluster/config.py

```diff
@@ -70,6 +72,8 @@ class ClusterConfiguration:
             The maximum amount of memory to allocate to each worker.
         num_gpus:
             The number of GPUs to allocate to each worker. (Deprecated, use worker_extended_resource_requests)
+        tolerations:
```

Comment:
```suggestion
        worker_tolerations:
```
Tested it in Kind and Openshift using taints work good.

Reply:
Thank you @Ygnas . Now it should be fixed.